proto/ethermint/evm/v1alpha1/query.proto

```diff
@@ -216,7 +216,10 @@ message QueryBlockLogsResponse {
 
 // QueryBlockBloomRequest is the request type for the Query/BlockBloom RPC
 // method.
-message QueryBlockBloomRequest {}
+message QueryBlockBloomRequest {
+  // height of the block which we want to query the bloom filter
+  int64 height = 1;
```

Comment:
can you add the note here? otherwise someone might revert the change by accident

Reply:
sure, added more comments and changelog